build.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
 # get/update the source code and build bellman-cuda library
-if cd bellman-cuda; then git pull; else git clone https://github.com/matter-labs/bellman-cuda.git --branch dev bellman-cuda; fi
-cmake -Bbellman-cuda/build -Sbellman-cuda/ -DCMAKE_BUILD_TYPE=Release
-cmake --build bellman-cuda/build/
+if cd era-bellman-cuda; then git pull; else git clone https://github.com/matter-labs/era-bellman-cuda.git --branch main era-bellman-cuda; fi
+cmake -Bera-bellman-cuda/build -Sbellman-cuda/ -DCMAKE_BUILD_TYPE=Release
```

Comment:
i think this should be `-Sera-bellman-cuda/`

Reply:
Thanks -- fixed!